### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by ngastelum-ms**

### CODE REVIEW

**Review by abhipsaMisra — APPROVED** [review]
😞

**Comment by abhipsaMisra** [comment]
You'll need to do this for the previews/v2 branch as well.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Track1 .NET Azure IoT Hub and DPS SDKs
 
-*	@drwill-ms @timtay-microsoft @abhipsaMisra @andyk-ms @brycewang-microsoft @tmahmood-microsoft @ngastelum-ms @patilsnr @schoims
+*	@drwill-ms @timtay-microsoft @abhipsaMisra @andyk-ms @brycewang-microsoft @tmahmood-microsoft @patilsnr @schoims
```
Comment (schoims) [inline .github/CODEOWNERS:3]: Could you remove me as well?

Comment (schoims) [inline .github/CODEOWNERS:3]:
```suggestion
*	@drwill-ms @timtay-microsoft @abhipsaMisra @andyk-ms @brycewang-microsoft @tmahmood-microsoft @patilsnr

**Review by drwill-ms — APPROVED** [review]

**Comment by ngastelum-ms** [comment]
> 😞
😞

**Pull request merged**